packages/sdk/test-next/messageStatus.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { describe, expect, it } from 'vitest'
```

Comment:
new file because old tests weren't running without .spec ?

Reply:
Correct without a .spec they were never running and we thought they were. Thank you so much for this fix!